### DISCUSSION TITLE
Preview a set of images

### DISCUSSION THREAD

**Discussion by jmsuresh** [quasarframework/quasar#13877] [General - Components / Directives / etc]
In social media like Twitter or Facebook, when the user posts more than one image, it shows a preview of these images as a collage. The user will be able to click on any of the images and maximize it.

I am trying to do similar thing in Quasar. Please let me know any pointers that could be of help.

Thanks,

**Comment by suleimanas** [comment]
this could help https://github.com/quasarframework/quasar/discussions/13776
